docker/build-common.sh

```diff
@@ -26,6 +26,12 @@ ${CARGO} ${CARGOFLAGS} build --release \
          -p backup-cli \
          "$@"
 
+# Build and overwrite the libra-node binary with feature failpoints if $ENABLE_FAILPOINTS is configured
+if [ "$ENABLE_FAILPOINTS" = "1" ]; then
+  echo "Building libra-node with failpoints feature"
+  (cd libra-node && ${CARGO} ${CARGOFLAGS} build --features failpoints "$@")
```

Comment:
(1) In step above we use `-p libra-node` instead of `cd`. I think its best to do things uniformly

(2) This line is missing `--release` flag, this is actually a big difference for libra-node. Probably need to introduce a flag like `${COMMON_CARGO_BUILD_ARGS}`, put `--release` there and use it here, below when building ct, and in line above.

Reply:
(1) doesn't work, we can't enable feature flag at the workspace root.
(2) good catch